Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Base/PaletteBase.cs

```diff
@@ -127,6 +128,33 @@ protected PaletteBase([DisallowNull] Image[] /*ICollection<Image>*/ buttonSpecIm
 
             _baseFontSize = 9f;
         }
+
+        /// <summary>Initializes a new instance of the <see cref="PaletteBase" /> class.</summary>
+        /// <param name="buttonSpecImageDictionary">The button spec image dictionary.</param>
+        protected PaletteBase([DisallowNull] IDictionary<PaletteButtonSpecStyle, Image> buttonSpecImageDictionary)
+        {
+            Debug.Assert(buttonSpecImageDictionary != null);
+
+            if (buttonSpecImageDictionary != null)
+            {
+                _buttonSpecImageDictionary = buttonSpecImageDictionary;
+            }
+
+            // We need to notice when system color settings change
+            SystemEvents.UserPreferenceChanged += OnUserPreferenceChanged;
+
+            // Inherit means we need to calculate the value next time it is requested
+            _dragFeedback = PaletteDragFeedback.Inherit;
+
+            _themeName = string.Empty;
+
+            _useKryptonFileDialogs = true;
+
+            _baseFont = _defaultFontStyle;
+
+            _baseFontSize = 9f;
```

Comment:
Why is this hardcoded, It should be from the default style

Reply:
Now dynamically set